src/collection/dimensions.js

```diff
@@ -266,6 +266,33 @@ fn = elesfn = ({
         };
       }
 
+      function computePaddingValues( width, height, paddingObject, relativeTo ) {
+        // Assuming percentage is number from 0 to 1
+        if(paddingObject.units === '%') {
+          return {
```

Comment:
Using a `switch` statement would increase performance here and it would make sense to use `default:`